app/views/posts/index.html.erb

```diff
@@ -9,13 +9,17 @@
       <p><%= post.text %></p>
       <div class='stats'>
         <span>Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></span>
+         <% if can? :destroy, post %>
+          <%= link_to "Delete Post", user_post_path(user_id: post.author_id, id: post.id), data: { turbo_method: :delete } %>
+        <% end %>
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/225777176-22d61362-5afa-46c9-942d-b6a7c20ef406.png)

- [ ] KIndly solve this error that occurs when a  comment is deleted